### DISCUSSION TITLE
Change in learning vm ip

### DISCUSSION THREAD

**Discussion by daveanvit** [Sunbird-Ed/Community#170] [Installation] [answered]
I had updated leaning VM ip Core/Common.yaml @keshavprasadms @saiakhil46 can please you help which jobs do i need to re-run?
is it will be same as this https://github.com/Sunbird-Ed/Community/discussions/149

**Comment by saiakhil46** [comment, marked as answer]
@daveanvit 
Deploy below core services:
1. content-service
2. assessment-service
3. taxonomy-service
4. knowledgemw-service
5. player

if not added please add the learning ip in below 

Add in knowledgeplatform/hosts and deploy below services.
1. learning-service
2. yarn jobs 
3. All flink jobs 

Add in datapipeline/hosts and deploy below services
1. eddataproducts
2. coredataproducts
3. Flink jobs --> content-cache-updater, user-cache-updater-v2, assessment-aggregator

**Reply by daveanvit** [reply]
okay, got it thanks @saiakhil46

**Reply by saiakhil46** [reply]
@daveanvit please use ip address for `learningservice_ip` variable

**Reply by riddhibatas-searce** [reply]
@saiakhil46 
I have updated azure storage account credentials, can you please let me know what services do I need to redeploy and do I need to rebuild any other pipeline?

**Reply by beepdot** [reply]
@riddhi2310 Please post new threads if the issue is not related to original issue.
